build/metadata.sh

```diff
@@ -21,6 +21,10 @@ RELATED_IMAGE_PROMETHEUS_WEBHOOK_SNMP=${RELATED_IMAGE_PROMETHEUS_WEBHOOK_SNMP:-q
 RELATED_IMAGE_PROMETHEUS_WEBHOOK_SNMP_TAG=${RELATED_IMAGE_PROMETHEUS_WEBHOOK_SNMP_TAG:-latest}
 RELATED_IMAGE_OAUTH_PROXY=${RELATED_IMAGE_OAUTH_PROXY:-quay.io/openshift/origin-oauth-proxy}
 RELATED_IMAGE_OAUTH_PROXY_TAG=${RELATED_IMAGE_OAUTH_PROXY_TAG:-latest}
+RELATED_IMAGE_PROMETHEUS=${RELATED_IMAGE_PROMETHEUS:-quay.io/prometheus/prometheus}
+RELATED_IMAGE_PROMETHEUS_TAG=${RELATED_IMAGE_PROMETHEUS_TAG:-latest}
+RELATED_IMAGE_ALERTMANAGER=${RELATED_IMAGE_ALERTMANAGER:-quay.io/prometheus/alertmanager}
+RELATED_IMAGE_ALERTMANAGER_TAG=${RELATED_IMAGE_ALERTMANAGER:-latest}
```

Comment:
We're using the alertmanager repo variable twice here, instead of the tag variable. Line 27 should be:
`RELATED_IMAGE_ALERTMANAGER_TAG=${RELATED_IMAGE_ALERTMANAGER_TAG:-latest}`

Reply:
Nice catch. I will fix this.